languages/javascript/concepts/equality/introduction.md

```diff
@@ -0,0 +1,66 @@
+**Equality** in the javascript can be referred in two ways which is `==` or `===`
+It is usually a misconception that people consider `==` which is usually check for value equality and `===` which usually check for the equality and type of the comparison which is not true.
+
+The usual description as define in the specification of the javascript is that
+`==` allows coercion in the equality while `===` disallow coercion
+
+`==` is usually defined as the abstract equality operator. **This equality prefers Numeric coercion**
```

Comment:
I would avoid "usually". That can be misleading to a reader as to wonder in which case is it "usually" defined and which case it is not "usually" defined.